docs/pages/kubernetes-access/discovery.mdx

```diff
@@ -0,0 +1,118 @@
+no longer meet the filtering labels.
+
+The following snippet describes the different configuration options for the Discovery Service 
+and their default values.
+
+```yaml
+# This section configures the Discovery Service
+discovery_service:
+    enabled: "yes"
+    aws:
+       # AWS resource types. Valid options are:
+       # eks - discovers and registers AWS EKS clusters
+       # ec2 - discovers and registers AWS EC2 Machines
+     - types: ["eks"]
+       # AWS regions to search for resources from
+       regions: ["eu-central-1"]
+       # AWS resource tags to match when registering resources
+       # Optional section: Defaults to "*":"*"
+       tags:
+         "env": "prod"
+    # Matchers for discovering Azure-hosted resources.
+    azure:
+      # Azure resource types. Valid options are:
+      # 'aks' - discovers and registers Azure AKS Kubernetes Clusters.
+    - types: ["aks"]
+      # Azure regions to search for resources from. Valid options are:
+      # '*' - discovers resources in all regions (default).
+      # Any valid Azure region name. List all valid regions using the Azure "az" cli: `az account list-locations -o table`
+      regions: ["*"]
+      # Azure subscription IDs to search resources from. Valid options are:
+      # '*' - discovers resources in all subscriptions (default).
+      # Any subscription_id: `az account subscription list -o table`
+      subscriptions: ["*"]
+      # Azure resource groups to search resources from. Valid options are:
+      # '*' - discovers resources in all resource groups within configured subscription(s) (default).
+      # Any resource_groups: `az group list -o table`
+      resource_groups: ["*"]
+      # Azure resource tag filters used to match resources.
+      # Optional section: Defaults to "*":"*"
+      tags:
+        "env": "prod"
+```
+
+### Forwarding requests to the Kubernetes Cluster
+
+The Teleport Kubernetes Service is responsible for monitoring the dynamic resources created or 
+updated by the Discovery Service and forwarding requests to the Kubernetes clusters they represent.
+To work correctly, it requires direct access to the target Kubernetes clusters and 
+permissions to forward requests.
+
+To turn on dynamic resource monitoring in the Kubernetes Service, you must configure 
+the `kubernetes_service.resources` section as shown in the following snippet:
+
+```yaml
+## This section configures the Kubernetes Service
+kubernetes_service:
+    enabled: "yes"
+    # Matchers for dynamic Kubernetes cluster resources created with the "tctl create" command or by Kubernetes auto-discovery.
+    resources:
+    - labels:
+        "*": "*" # can be configured to limit the clusters to watched by this service.
+```
+
+Both services — Discovery and Kubernetes — can be configured in the same 
+Teleport process or separate processes. 
+When they run separately, you need to take care of the permissions required for each service.
```

Comment:
Nit: I would either remove this sentence or expanded on this, right now it's not clear what exactly this asks user to do.